tax-documents.md

```diff
@@ -0,0 +1,38 @@
+# Tax Documents Needed for Tech Setup
+You will need three kinds of tax documents to get setup in most systems.
+
+## IRS Determination Letter
+Your PTA Treasurer and/or Secretary should have a copy of the [IRS determination letter](./docs/irs-det-txptacongress.pdf).  This is a letter from the IRS containing the PTA Texas Congress group exemption #1889.  The packet should contain two documents in total, and is proof that your PTA is a non-profit organization.
+
+1. A letter from the IRS which contains text substancially similar to: "Your organization is exempt under section 501 (c)(3) of the Code because it is included in a group ruling issued to PTA Texas Congress, located in Austin, TX."
```

Comment:
```suggestion
1. A letter from the IRS which contains text substantially similar to: "Your organization is exempt under section 501 (c)(3) of the Code because it is included in a group ruling issued to PTA Texas Congress, located in Austin, TX."
```